packages/ui/src/components/va-checkbox/_variables.scss

```
@@ -17,4 +17,10 @@
   --va-checkbox-square-height: 1.25rem;
   --va-checkbox-square-border: solid var(--va-form-element-border-width) var(--va-background-border);
   --va-checkbox-square-border-radius: calc(var(--va-form-element-border-radius) / 2);
+
+  /* Padding */
+  --va-checkbox-vertical-padding: 0.3rem 0 0.3rem 0.5rem;
+  --va-checkbox-horizontal-padding: 0 0.5rem;
+  --va-checkbox-right-padding: 0 0.5rem 0 0;
+  --va-checkbox-left-padding: 0 0 0 0.5rem;
```

Can we have something like this? I'm not sure how to name 0.3rem though. Do you have any ideas?

Maybe vertical-gap and horizontal-gap.

```suggestion
  --va-checkbox-gap: 0.5rem;
  --va-checkbox-vertical-padding: 0.3rem 0 0.3rem var(--va-checkbox-gap);
  --va-checkbox-horizontal-padding: 0 var(--va-checkbox-gap);
  --va-checkbox-right-padding: 0 var(--va-checkbox-gap) 0 0;
  --va-checkbox-left-padding: 0 0 0 var(--va-checkbox-gap);
```

---

I would be much better if we were applying only paddingLeft and paddingRight instead of making them 0px every time this with that we could have directly give `vertical-gap` and `horizontal-gap` as values. But they might be required to overwrite anything.

So,  let's just go with `horizontal-gap` and `vertical gap`